sdk/core/core-http/lib/serviceClient.ts

```diff
@@ -593,6 +593,86 @@ function createDefaultRequestPolicyFactories(
   return factories;
 }
 
+export function createPipelineFromOptions(
+  pipelineOptions: PipelineOptions,
+  authPolicyFactory?: RequestPolicyFactory
+) : ServiceClientOptions {
```

Comment:
What is being returned is `ServiceClientOptions`. Then why is the name of this function `createPipelineFromOptions`?

Reply:
Mainly because of the naming that we've been using in KeyVault and Storage libraries, they rename `ServiceClientOptions` to `Pipeline` on import.  We should definitely discuss the proper naming for this (and also the rename/rexport being done for `ServiceClientOptions` in KeyVault libraries).